sim_pipeline/image_simulation.py

```diff
@@ -71,17 +71,17 @@ def gsobj(image, pix_scale, flux):
     :param pix_scale: pixel scale to be asigned to the interpolated image
     :param flux: flux value to be asigned to the interpolated image
     :returns : interpolated image with specified pixel scale and flux"""
-    gso = galsim.InterpolatedImage(galsim.Image(image), scale = pix_scale, flux = flux)
-    return gso
+    # gso = galsim.InterpolatedImage(galsim.Image(image), scale = pix_scale, flux = flux)
+    # return gso
 
 def gsobj_true_flux(image, pix_scale):
     """
     :param image: image that need to be interpolated
     :param pix_scale: pixel scale to be asigned to the interpolated image
     :param flux: flux value to be asigned to the interpolated image
     :returns : interpolated image with specified pixel scale and flux"""
-    gso_test = galsim.InterpolatedImage(galsim.Image(image), scale = pix_scale, normalization='flux')
```

Comment:
this function is required for the RSP source injection. So galsim is somehow required